### DISCUSSION TITLE
Azure Verified Modules

### DISCUSSION THREAD

**Discussion by mark-gronow** [aztfmod/terraform-azurerm-caf#1911] [Ideas]
I see Microsoft are using [Azure Verified Modules](https://azure.github.io/Azure-Verified-Modules/) for aligning consistent coding patterns across Bicep and Terraform with MS WAF best practice configurations. 

If not already could the same be done for SRE Terraform?  It would be great to know that the code does follow the same patterns or is able to consume the Azure Verified Modules. Something we can let our customers know when deploying it.

**Comment by darrenhull** [comment]
Is the real question, what is the future of this this module if Microsoft are creating [Azure Verified Modules](https://azure.github.io/Azure-Verified-Modules/) will this module use those modules as the provider?
